src/graphql/schema/buildMutationInputType.ts

```diff
@@ -68,7 +68,7 @@ function buildMutationInputType(name: string, fields: Field[], parentName: strin
       let type: PayloadGraphQLRelationshipType = GraphQLString;
 
       if (Array.isArray(relationTo)) {
-        const fullName = `${combineParentName(parentName, field.label)}RelationshipInput`;
+        const fullName = `${combineParentName(parentName, !field.label ? '' : field.label)}RelationshipInput`;
```

Comment:
At this point, this is where GraphQL types are being named. We should be using the sanitized label for the field here. Using an empty string will create the potential for naming conflicts.